### DISCUSSION TITLE
errors in faces module

### DISCUSSION THREAD

**Discussion by mariannevanharten** [fisharebest/webtrees#4728] [General]
In the website logs at my family tree in webtrees I see a lot of strange items, such as
[code]SQLSTATE[HY000]: General error: 1390 Prepared statement contains too many placeholders (SQL: select `i_id` as `xref`, `i_gedcom` as `gedcom` from `wt_individuals` where `i_file` = 16 and `i_id` in (I19950, I20979, I48831, <continues a long list of ID's>
[/code]

Best regards,
Marianne

**Comment by fisharebest** [comment]
This needs to be in the faces issue tracker - not the webtrees one.  https://github.com/UksusoFF/webtrees-faces/issues

I don't think UksusoFF will see it here...

**Reply by mariannevanharten** [reply]
OOPS! Thank you!
